package.json

```diff
@@ -58,7 +58,7 @@
     "openfin-service-signal": "^1.0.0",
     "openfin-service-tooling": "1.0.17",
     "pre-commit": "^1.2.2",
-    "puppeteer": "1.3.0",
+    "puppeteer": "1.17.0",
```

Comment:
Good job on the fixed versioning! However `~1.17.0` might be best

Reply:
They're already onto 1.19, so I doubt there would ever be a minor version release on .17 (and if there is, we would probably want to update it manually anyway). I think single specific version is fine here.